src/terrain/terrain.js

```diff
@@ -587,7 +588,8 @@ export class Terrain extends Elevation {
             useDepthForOcclusion?: boolean,
             useMeterToDem?: boolean,
             labelPlaneMatrixInv?: ?Float32Array,
-            morphing?: { srcDemTile: Tile, dstDemTile: Tile, phase: number }
+            morphing?: { srcDemTile: Tile, dstDemTile: Tile, phase: number },
+            denormalize?: boolean
```

Comment:
Should we use some more descriptive name for the parameter?

Reply:
sure. How about `useDenormalizedUpVector`?